variants/rakwireless_rak11300/pins_arduino.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+// Pin definitions taken from:
+//    RAK definition: https://github.com/RAKWireless/RAK-RP-Arduino/
+//    RAK datasheet: https://docs.rakwireless.com/Product-Categories/WisDuo/RAK11300-Module/Datasheet/#overview
+//    Internal wiring of SX1262 module: https://forum.rakwireless.com/t/rak11300-pinout-rp2040-to-sx1262/8414/
```

Comment:
Maybe we can add the pin definitions for SX1262 NRESET, BUSY, DIO1 and ANT_PWR (GPIO25) directly in this file, together with a define for SX1262_SPI = SP1?

Reply:
Thanks, I did most of that and added some more comments as some of these pins have multiple names and this might be otherwise confusing. I kept SPI1 on SPI1 as it neatly integrates with the RadioLib library.